src/code/ucode_disas.c

```diff
@@ -60,14 +64,17 @@ const char* UCodeDisas_ParseCombineColor(u32 value, u32 idx) {
         case G_CCMUX_ENVIRONMENT:
             ret = "ENVIRONMENT";
             break;
-        case 6:
-            ret = (idx == 2) ? "CENTER" : (idx == 3) ? "SCALE" : "1";
+        case 6: // G_CCMUX_CENTER, G_CCMUX_SCALE, G_CCMUX_1
+            ret = (arg == COMBINER_B) ? "CENTER" : (arg == COMBINER_C) ? "SCALE" : "1";
             break;
-        case 7:
-            ret = (idx == 1) ? "NOISE" : (idx == 2) ? "K4" : (idx == 3) ? "COMBINED_ALPHA" : "0";
+        case 7: // G_CCMUX_NOISE, G_CCMUX_K4, G_CCMUX_COMBINED_ALPHA, G_CCMUX_0
+            ret = (arg == COMBINER_A)   ? "NOISE"
+                  : (arg == COMBINER_B) ? "K4"
+                  : (arg == COMBINER_C) ? "COMBINED_ALPHA"
+                                        : "0";
             break;
         default:
```

Comment:
```suggestion
            break;
            
        default:
```
Could you add some new lines between the breaks and cases? Applies to the other switches as well.